### ISSUE TITLE
An error occurred: Timeout

### ISSUE THREAD

**Issue by laggis** [jaimeadf/discord-screenshot#20]
Hey so my discord-screenshot just stop working randomly i have no clue why, im getting "An error occurred: Timeout" in the discord

![Untitled](https://user-images.githubusercontent.com/18127838/116799920-ce075680-aafc-11eb-94e8-15476330b203.png)

**Comment by jaimeadf** [comment]
This happens when the screenshot upload takes more than 1 minute, which usually means that the player has slow internet or your server is having bandwidth problems. If you restart your server does it work normally?

**Comment by laggis** [comment]
If you mean normally like if you can connect throw and stuff like that yes that works everything works fine, just that im getting that timeout thing and my internet its not that bad so maybe a port blocking it?

Update it works perfect before but for some odd reason it just stop working, do i have to have a order in the server.cfg with the "discord-screenshot" "screenshot-basic"

**Comment by jaimeadf** [comment]
I was referring to the script itself, but if the problem was a blocked port, it wouldn't even work. Does that happen to all your connected players or only yourself?

**Comment by laggis** [comment]
Yes, and i have no error comming from the scrip

**Comment by jaimeadf** [comment]
All players or just you?

**Comment by laggis** [comment]
All

**Comment by jaimeadf** [comment]
What host provider are you using?
Have you taken any screenshots before the error, if so, how many?

Sorry for making so many questions.

**Comment by laggis** [comment]
Im using zaphosting, 

Yes i was enable to take a screenshots before, like 10+ 

You can ask how many you want bc i have tried everything i can understand why

**Comment by laggis** [comment]
When i join the server and took a screenshot this come up im not sure if it means anything

![image](https://user-images.githubusercontent.com/18127838/116800982-1ecf7d00-ab06-11eb-892d-b4b12345da06.png)
![image](https://user-images.githubusercontent.com/18127838/116800993-33137a00-ab06-11eb-9ca8-81540d6aef75.png)

**Comment by laggis** [comment]
Gettings this problem now


![Untitled](https://user-images.githubusercontent.com/18127838/116842280-485ad800-abdc-11eb-938d-ce67ee82125b.png)

**Comment by jaimeadf** [comment]
Both warnings are from screenshot-basic and I can't do anything about them. If you want to see more details about the last one take a look at the issue #18.

**Comment by jaimeadf** [comment]
Could you try restarting both resources (`screenshot-basic` first and then `discord-screenshot`) and test if they work?

**Comment by laggis** [comment]
Im just getting this now


![116800982-1ecf7d00-ab06-11eb-892d-b4b12345da06](https://user-images.githubusercontent.com/18127838/117029020-8cde9480-acfe-11eb-9bce-fa1d5dbe2bdb.png)
![116800993-33137a00-ab06-11eb-9ca8-81540d6aef75](https://user-images.githubusercontent.com/18127838/117029023-8d772b00-acfe-11eb-8764-61a35d35d4e4.png)

**Comment by jaimeadf** [comment]
> Both warnings are from screenshot-basic and I can't do anything about them. If you want to see more details about the last one take a look at the issue #18.

^^

**Comment by jaimeadf** [comment]
Are you using the `screenshot-basic` that cames with `discord-screenshot`?

**Comment by laggis** [comment]
Yes, but then im just getting the "(node:2052) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead." 

I have check the links you gave me going to read more about when i have time
